### PR TITLE
 new: Add support for Parent/Child account switching

### PR DESCRIPTION
**Pull request by jriddle-linode**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -121,3 +121,6 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+// TODO: Remove before merging into dev branch
+replace github.com/linode/linodego => github.com/linode/linodego v1.29.1-0.20240306151024-1a81a0de8122
```
Comment (lgarber-akamai) [inline go.mod:126]: Just a heads up we should remove this before merging 👍

**Comment by jriddle-linode** [comment]
jobs failing until Linodego release

**Comment by lgarber-akamai** [comment]
@jriddle-linode Linodego release was just cut, could you bump the go.mod when you get a chance?

**Review by zliang-akamai — APPROVED** [review]

**Review by lgarber-akamai — APPROVED** [review]
LGTM, thanks for addressing the merge conflicts!

**Pull request merged**